pkg/cmd/server/bootstrappolicy/all_test.go

```diff
@@ -16,10 +16,7 @@ const osClusterRoleAggregationPrefix = "system:openshift:"
 // this map must be manually kept up to date as we make changes to aggregation
 // we hard code this data with no constants because we cannot change the underlying values
 var expectedAggregationMap = map[string]sets.String{
-	"admin":          sets.NewString("system:openshift:aggregate-to-admin", "system:aggregate-to-admin"),
-	"edit":           sets.NewString("system:openshift:aggregate-to-edit", "system:aggregate-to-edit"),
-	"view":           sets.NewString("system:openshift:aggregate-to-view", "system:aggregate-to-view"),
-	"cluster-reader": sets.NewString("system:openshift:aggregate-to-view", "system:aggregate-to-view", "system:openshift:aggregate-to-cluster-reader"),
+	"cluster-reader": sets.NewString("registry-viewer", "system:openshift:aggregate-to-view", "system:openshift:aggregate-to-cluster-reader"),
```

Comment:
This test is annoying.

Reply:
It does not make sense unless you have both the OS and Kube RBAC default cluster roles (since you cannot see what we are aggregating where).

Reply:
> It does not make sense unless you have both the OS and Kube RBAC default cluster roles (since you cannot see what we are aggregating where).

I agree. Since those are logically in two separate repos, let's just delete it.